nss-day-cohort-60/national-parks-django-api
# Sc campground views

Adding Gets to CampgroundView

To test --
git fetch origin sc-campground-views
git checkout sc-campground-views

start or refresh server

you can check data via postman or in the client 

if using postman, 

to GET all campgrounds 
send GET request to 
http://localhost:8000/campgrounds

see 200 response and returned data

to GET one campground by ID 
http://localhost:8000/campgrounds/1

see 200 response and returned data

to GET campgrounds by park_id 
http://localhost:8000/campgrounds?park_id=6

see 200 response and returned data


In the client

refresh your client with the server running

navigate to a single park

scroll down to campgrounds

see campgrounds rendered in individual parks
